lib/classes/genre.rb

```diff
@@ -1,5 +1,30 @@
 class Genre
-  def initialize(sample)
-    @sample = sample
+  attr_accessor :items
+  attr_reader :id, :name
+
+  def initialize(name, id = rand(1..1000))
+    @name = name
+    @id = id
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item.genre = self
+  end
+
+  def to_json(*args)
+    {
+      JSON.create_id => self.class.name,
+      'name' => name,
+      'id' => id,
+      'items' => @items
+    }.to_json(*args)
+  end
+
+  def self.json_create(object)
+    obj1 = new(object['name'], object['id'])
+    obj1.items = object['items']
+    obj1
   end
```

Comment:
Awesome!! I loved the idea of having the ```to_json```  and ```json_create ``` methods inside the class, I don't know if you have already tried this (I guess you do), kindly let me know to ask other team members to implement these methods and have a consistent codebase

Reply:
@Trictonicmp  Yeah I used this method in my last project, I didn't find any better before, but I am open to any suggestion